pom.xml

```diff
@@ -113,13 +112,6 @@
       <scope>runtime</scope>
     </dependency>
 
-    <dependency>
-      <groupId>com.activeviam.tech</groupId>
-      <artifactId>content-server-ui</artifactId>
-      <version>${content-server-ui.version}</version>
-      <scope>runtime</scope>
-    </dependency>
```

Comment:
why we don't need the ui of the content server ?

Reply:
by default MAC does not use the content server ui

Reply:
Since 5.11 we must init the content server through the content ui. Is this done somewhere in the code which remove the burden of doing it on the content ui ?

Reply:
this is done by loading the bookmarks. and the loading of the bookmarks is done if no bookmarks exist (like the first time you start MAC)